assignments/4. todo/todo/main.js

```diff
@@ -0,0 +1,211 @@
+const todoItemsStorageKey = "todo-app-items";
+const viewModeStorageKey = "todo-app-view-mode";
+let todoItems = JSON.parse(window.localStorage.getItem(todoItemsStorageKey)) || [];
+let viewMode = window.localStorage.getItem(viewModeStorageKey) || "all"; // possible values are all, active and completed
+
+let todoAutoIndex = Date.now();
+
+const form = document.getElementById("input-form");
+form.addEventListener("submit", event => {
+	event.preventDefault();
+	const inputBox = document.getElementById("input-box");
+	const text = inputBox.value.trim();
+	if (text != "") {
```

Comment:
Equivalent. Using `!` makes it hard to think sometimes. Try to keep things simple if you can.
```js
if(text){
}
```